src/app/index.tsx

```diff
@@ -8,49 +8,56 @@ import {
   Image,
   Animated,
   useAnimatedValue,
+  TouchableOpacity,
+  TextInput,
 } from "react-native";
 import { useSafeAreaInsets } from "react-native-safe-area-context";
+import Loading from "../components/screens/loading";
+import Home from "../components/screens/home";
+import AddApp from "../components/screens/addapp";
+import AppDetail from "../components/screens/addedappid";
+import { init, i, InstaQLEntity, id } from "@instantdb/react-native";
+import db from "./db";
 
-const panther = require("../media/panther.jpg");
-// const [theme, setTheme] = useState("light");
-
-function loading() {
+export default function App() {
+  const panther = require("../media/panther.jpg");
   const [page, setPage] = useState("load");
+  const [apps, setApps] = useState([]);
 
-  const textSlide = useRef(new Animated.Value(-500)).current;
-  useEffect(() => {
-    Animated.timing(textSlide, {
-      toValue: 0,
-      duration: 2000,
-      useNativeDriver: true,
-    }).start();
-  }, [textSlide]);
+  if (page.startsWith("App_")) {
+    const parts = page.split("_");
+    const appData = {
+      appId: parts[1],
+      appName: parts[2],
```

Comment:
You don't really need appName, appDesc. 

You can get this by writing a custom query:

```
const { isLoading, error, data} = db.useQuery({ appslist: { $:  { where: { id: '...' });
if (...) return
// data.appslist[0] -- this will be your app!
```